peekingduck/nodes/draw/utils/bbox.py

```diff
@@ -125,6 +141,41 @@ def _draw_label(
     )
 
 
+# draw score function
+def _draw_score(
+    frame: np.ndarray,
+    bottom_left: Tuple[int, int],
+    bbox_score_str: str,
+    bg_color: Tuple[int, int, int],
+    text_color: Tuple[int, int, int],
+) -> None:
+    """Draws prediction score at bottom left of bbox."""
+    # get label size
+    (text_width, text_height), baseline = cv2.getTextSize(
+        bbox_score_str, FONT_HERSHEY_SIMPLEX, NORMAL_FONTSCALE, THICK
+    )
+    # put filled text rectangle
+    cv2.rectangle(
+        frame,
+        (bottom_left[0], bottom_left[1]),
+        (bottom_left[0] + text_width, bottom_left[1] - text_height - baseline),
+        bg_color,
+        FILLED,
+    )
+
+    # put text above rectangle
+    cv2.putText(
+        frame,
+        bbox_score_str,
+        (bottom_left[0], bottom_left[1] - 6),
+        FONT_HERSHEY_SIMPLEX,
+        NORMAL_FONTSCALE,
+        text_color,
+        THICK,
+        LINE_AA,
+    )
```

Comment:
Instead of hardcoding `top-left` for label, `bottom-left` for score, it would be much better to allow user to specify label and score positions.
Can stick to four positions: `top-left`, `bottom-left`, `top-right`, `bottom-right`.
Write a function to return the correct set of coords based on the four positions.

Reply:
Created a separate [issue](https://github.com/aisingapore/PeekingDuck-Private/issues/116) for tracking